tests/qchem/test_basis_set.py

```diff
@@ -397,6 +399,33 @@ def test_atom_basis_data(self, basis_name, atom_name, params_ref):
         assert alpha == alpha_ref
         assert coeff == coeff_ref
 
+    @pytest.mark.parametrize(
```

Comment:
Throughout these tests, it may be good to add one more "weird" example, e.g. for a less popular basis set and less common element.

Reply:
Added `6-31+G*` for `B`.